environments/openstack/playbook-bootstrap-basic.yml

```diff
@@ -203,10 +211,23 @@
         disk_format: qcow2
         filename: /tmp/ubuntu.img
         min_disk: 3
+        min_ram: 512
         properties:
           cpu_arch: x86_64
           distro: ubuntu
           hw_rng_model: virtio
+          architecture: x86_64
+          hypervisor_type: qemu
+          os_version: "20.04"
+          os_distro: ubuntu
+          hw_disk_bus: virtio
```

Comment:
Don't we want to default to scsi?
I think that is the default on most clouds and I believe that with respect to features and performance, scsi works at least as well as virtio.
So I would have expected
 `hw_disk_bus: scsi`
 `hw_scsi_model: virtio-scsi`